### PR TITLE
Scalability Test and Cluster Management Doc

### PR DESCRIPTION
**Pull request by hanahmily**
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking-banyandb/blob/main/CHANGES.md).

### CODE REVIEW

File: docs/operation/cluster.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Cluster Maintenance
```
Comment (wankai123) [inline docs/operation/cluster.md:1]: Missing the menu config?

File: docs/operation/cluster.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+The new added data nodes can be automatically discovered by the existing liaison nodes. It is recommended to add data nodes one by one to avoid overloading the liaison nodes with the new data nodes' metadata.
+
+The cluster's availability is also improved by increasing the number of data nodes, as active data nodes need to handle a lower additional workload when some data nodes become unavailable. For example, if one node out of 2 nodes is unavailable, then 50% of the load is re-distributed across the remaining node, resulting in a 100% per-node workload increase. If one node out of 10 nodes is unavailable, then 10% of the load is re-distributed across the 9 remaining nodes, resulting in only an 11% per-node workload increase.
+
+Increasing the number of etcd nodes can increase the cluster's metadata capacity and improve the cluster's metadata query performance. It can also improve the cluster's metadata availability, as the metadata is replicated across all the etcd nodes. However, the cluster size should be odd to avoid split-brain situations.
```
[the remark this answers is not in the document]
Comment (hanahmily) [reply docs/operation/cluster.md:57]: Absolutely.

During the test, I used 10 data nodes and only 1 etcd node in a medium-sized cluster. Moving forward, we need to include more extensive and complex scenarios in the scale testing.

[the remark this answers is not in the document]
Comment (wu-sheng) [reply docs/operation/cluster.md:57]: Yes, and when we meet Lenovo team next month, we need to verify the scale with them.

**Comment by wu-sheng** [comment]
Others are good, please fix menu structure for operation docs.

File: docs/operation/cluster.md (hunk 1 above)
Comment (hanahmily) [reply docs/operation/cluster.md:1]: done

**Comment by wu-sheng** [comment]
Why your yaml is so different?

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**